code/modules/materials/definitions/solids/materials_solid_butchery.dm

```diff
@@ -0,0 +1,226 @@
+/decl/material/solid/organic/meat
```

Comment:
this needs to be readded to `/decl/material/solid/organic/meat/adjust_nutrition()` or something
```
	var/malus_level = M.GetTraitLevel(/decl/trait/malus/animal_protein)
	var/malus_factor = malus_level ? malus_level * 0.25 : 0
	M.take_damage(TOX, removed * malus_factor)
	M.adjust_nutrition(nutriment_factor * removed * (1 - malus_factor))
```